_migration-assistant/migration-phases/index.md

```diff
@@ -1,23 +1,118 @@
 ---
 layout: default
 title: Migration phases
-nav_order: 30
+parent: Migration Assistant for OpenSearch
+nav_order: 40
+nav_exclude: false
 has_children: true
 has_toc: false
-permalink: /migration-phases/
-redirect_from: 
-  - /migration-phases/index/
+permalink: /migration-assistant/migration-phases/
+redirect_from:
+  - /migration-assistant/overview/migration-phases/
+  - /migration-phases/
 ---
 
 # Migration phases
 
-This page details how to conduct a migration with Migration Assistant. It shows you how to [plan for your migration]({{site.url}}{{site.baseurl}}/migration-assistant/migration-phases/planning-your-migration/index/) and encompasses a variety of migration scenarios, including:
+This page outlines the core phases of migrating with Migration Assistant. There are three migration scenarios, each consisting of a sequence of common steps.
 
-- [**Metadata migration**]({{site.url}}{{site.baseurl}}/migration-assistant/migration-phases/migrating-metadata/): Migrating cluster metadata, such as index settings, aliases, and templates.
-- [**Backfill migration**]({{site.url}}{{site.baseurl}}/migration-assistant/migration-phases/backfill/): Migrating existing or historical data from a source to a target cluster.
-- [**Live traffic migration**]({{site.url}}{{site.baseurl}}/migration-assistant/migration-phases/using-traffic-replayer/): Replicating live ongoing traffic from [a source cluster]({{site.url}}{{site.baseurl}}/migration-assistant/migration-phases/switching-traffic-from-the-source-cluster/) to a target cluster.
+<style>
```

Comment:
These panels are great! However, we do not put css directly into markdown pages. It makes the markdown page cluttered and it makes it much harder for a non-technical tech writer to edit the contents. All css should go into https://github.com/opensearch-project/documentation-website/blob/main/_sass/custom/custom.scss. Consider adding an html include for this panel (similar to how we do cards) that will take a title and an array of steps (with substeps). This way, the markdown is clean, you can specify the parameters in the front matter and specify the include in the markdown.